src/metabase/models/serialization.clj

```diff
@@ -1379,7 +1392,11 @@
 (defn- export-viz-click-behavior [settings]
   (some-> settings
           (m/update-existing    :click_behavior export-viz-click-behavior-link)
-          (m/update-existing-in [:click_behavior :parameterMapping] export-viz-click-behavior-mappings)))
+          (m/update-existing-in [:click_behavior :parameterMapping] export-viz-click-behavior-mappings)
+          (as-> updated-settings
+                (if (nil? (:click_behavior updated-settings))
+                  (dissoc updated-settings :click_behavior)
+                  updated-settings))))
```

Comment:
```suggestion
          (cond-> updated-settings
            (nil? (:click_behavior updated-settings)) (dissoc :click_behavior))
```

Reply:
I committed this but then realized it doesn't quite work, I ended up needing to wrap this back into `as->`.